holidays/mixins/child_entity.py

```
@@ -0,0 +1,31 @@
+
+    parent_entity: type[HolidayBase]
+    """The parent entity class."""
+
+    subdivisions: tuple[str, ...] = ()
+    """Override parent subdivisions."""
+
+    subdivisions_aliases: dict[str, str] = {}
+    """Override parent subdivisions aliases."""
+
+    def __init__(self, *args, **kwargs):
+        kwargs["subdiv"] = self.country
```

Pylint (3.3.7)

[error] Instance of 'ChildEntity' has no 'country' member

(E1101)